masonry/src/widget/scroll_bar.rs

```diff
@@ -162,7 +162,7 @@ impl Widget for ScrollBar {
             }
             PointerEvent::PointerUp(_, _) => {
                 self.grab_anchor = None;
-                ctx.set_active(false);
+                ctx.release_pointer();
```

Comment:
Oh, this reminds me. PointerUp (and PointerLeave) events should automatically release pointer capture at the pass level. I forgot to implement it initially.

I'd appreciate if you included it in your PR, but it's optional.

Reply:
I've included this, and added a test for `PointerLeave`. `PointerUp` releasing the pointer was already tested in https://github.com/linebender/xilem/blob/6c4951635da34cf04632332114e7b04ca343d8c0/masonry/src/widget/tests/status_change.rs#L205